hw-7/controllers/users.controller.test.ts

```diff
@@ -1,5 +1,163 @@
-describe('test', () => {
-  it('should pass', () => {
-    expect(true).toBe(true);
+import {User} from '@app-models/user.interface';
+import {Request, Response} from 'express';
+
+import {HTTP_ERROR} from '../constants/http-errors.enum';
+import UserFakeRepository from '../data-access/repositories/user-fake.repository';
+import UsersService from '../services/users.service';
+
+import {UsersController} from './users.controller';
+
+jest.mock('../services/users.service');
+
+describe('Users Controller', () => {
+  let service: UsersService;
+  let controller: UsersController;
+  let responseMock: Response;
+
+  beforeEach(() => {
+    service = new UsersService(new UserFakeRepository());
+    controller = new UsersController(service);
+    responseMock = getMockedResponse();
   });
-});
\ No newline at end of file
+
+  describe('getUsers()', () => {
```

Comment:
We could unify the `describe` of the `getUsers()` and use more specific test cases in each `it` test case.

Reply:
Thanks for the suggestion! Done.